concepticondata/conceptlists/Lynott-2009-423.tsv

```diff
@@ -0,0 +1,424 @@
+Lynott-2009-423-61	61	cheesy			1	3.04761904761905	0.476190476190476	0.666666666666667	4.19047619047619	4.71428571428571	1.65759435557046	0.980767435177556	1.31656117720877	1.07791686224159	0.560611910581388	Gustatory	0.323636363636364	0	0	6
+Lynott-2009-423-62	62	chequered			0.904761904761905	4.47368421052632	0.263157894736842	0.105263157894737	0	0	0.964274111134126	0.561951486949016	0.315301767642306	0	0	Visual	0.923913043478261	1	0.693147180559945	9
+Lynott-2009-423-63	63	chewy			1	2.57142857142857	2.04761904761905	1.33333333333333	0.619047619047619	4.47619047619048	1.6604646509766	1.80211515936664	1.39044357430761	1.07126982951031	1.40068010691405	Gustatory	0.349137931034483	0	0	5
+Lynott-2009-423-64	64	chilly			1	2.95238095238095	4.19047619047619	0.666666666666667	0.571428571428571	1.28571428571429	1.35927151357595	1.16700675315302	1.31656117720877	1.02817452659695	1.90112748501665	Haptic	0.374384236453202	4	1.6094379124341	6
+Lynott-2009-423-65	65	chocolatey			1	4.19047619047619	1	0.333333333333333	4.52380952380952	4.95238095238095	1.16700675315302	1.26491106406735	1.06458129484475	0.749603069567329	0.218217890235994	Gustatory	0.307936507936508	0	0	9
+Lynott-2009-423-66	66	chubby			1	4.95238095238095	4.09523809523809	0.428571428571429	0.571428571428571	0.19047619047619	0.218217890235994	1.22085060121056	1.07570574840095	1.32557265253281	0.679635756787974	Visual	0.465116279069767	1	0.693147180559945	6
+Lynott-2009-423-67	67	circular	CIRCULAR (ROUND IN TWO DIMENSIONS)	2302	1	4.76190476190476	3.85714285714286	0.142857142857143	0	0.095238095238095	0.538958431120796	1.42427926635594	0.358568582800318	0	0.436435780471985	Visual	0.537634408602151	19	2.99573227355399	8
+Lynott-2009-423-68	68	citrusy			1	2.57142857142857	0.571428571428571	0.047619047619048	3.95238095238095	5	1.46872150428284	0.978336781043653	0.218217890235992	0.804747816162957	0	Gustatory	0.407843137254902	0	0	7
+Lynott-2009-423-69	69	clammy			0.952380952380952	2.75	4.35	0.4	1.45	0.6	1.61815359364665	1.38696943388321	0.820782681668123	1.73129096949433	1.14248114115496	Haptic	0.413612565445026	1	0.693147180559945	6
+Lynott-2009-423-70	70	clamorous			0.380952380952381	2.25	2.875	2.375	0.875	0.125	1.48804761828569	2.16712449375401	2.32609421256197	1.35620268186054	0.353553390593274	Haptic	0.323529411764706	0	0	9
+Lynott-2009-423-71	71	clean	CLEAN	704	1	4.61904761904762	3.33333333333333	0.571428571428571	3.80952380952381	1.80952380952381	0.804747816162957	1.59164485150844	0.978336781043653	1.28914885172534	1.77817455889594	Visual	0.286195286195286	64	4.17438726989564	5
+Lynott-2009-423-72	72	clear	CLEAR	989	1	4.52380952380952	1.23809523809524	2.52380952380952	1.04761904761905	0.571428571428571	0.813575295780767	1.64026710949046	2.06443812256623	1.53219419383414	1.20712172424443	Visual	0.399038461538462	255	5.54517744447956	5
```

Comment:
Our CLEAR definition is aimed towards the sense of 'clear mind', 'clear ideas', i.e. it says 'free of ambiguity or doubt'. This clear here is probably more aimed at 'clear glass' etc.?

Reply:
The concepts in this list are sorted by ontological field, so it does mean clear in the sense of clear mind.

Reply:
But this clear in the list also has the addition of 'visual'?

Reply:
Or might not, confused that with another recent list.

Reply:
I've deleted the mapping.

Reply:
When discussing definitions in concepticon and the mapping of a concept-set to a gloss, it is also important to check at what is mapped so far. So the starting point would, beyond the definition for Concepticon CLEAR also the elicitation glosses it has been mapped to, and I would bet that this links to the visible clear, similar to the primary meaning of German "klar".

Reply:
One might guess so, but the already mapped ones all seem to mean clear to the mind, clear to understand, and similar.

Reply:
I do not agree. Most of them are ambiguous, and specifrically the Chinese concepts allow for both interpretations. Furthermore, Buck is ambiguous, providing both, one labelled "clear", one "plain", and the follow-up lists re-interprete this towards the same neutral term that includes clear as a visual description AND clear as a metaphorical extension. So clear is both visual and mental. And if there's only one clear in this list, I'd just link it to clear and keep it like this, until we get a list that actually makes a distinction here that is clearly mental or visual.

Reply:
But isn't the focus of this list exactly to have sensory adjectives, i.e. visual, haptic, auditory, olfactory and gustatory concepts?

Reply:
Why would Buck be ambiguous? It has "clear (adj., vs. obscure)", not just "clear". In the book itself, the entry is called "CLEAR, PLAIN (To the Mind)", and that's also what the text below it talks about. Borin and Haspelmath also have only example sentences that use the mental meaning - not like they are reliable, but they are there at least. One of the two Chinese glosses seemed to disambiguate it via "讲得清楚" but the other ones doesn't, so I guess you're right that they are ambiguous.

> But isn't the focus of this list exactly to have sensory adjectives, i.e. visual, haptic, auditory, olfactory and gustatory concepts?

If that's the case this _clear_ here should be left unmapped I suppose.

Reply:
Given that we have ambiguous cases, clear should then be split into two or three: CLEAR (visual or mental), CLEAR (visual) and CLEAR (mental). If you search for clear in [concepticon/values](https://concepticon.clld.org/values), you can also see that there are more lists which could then perhaps be more properly mapped. We can leave the CLEAR as the ambiguous part: Buck is ambiguous, since "obscure" also has two meanings, and the prominent one is "undurchsichtig" which has also two meanings, etc. So most of the data have CLEAR (visual or mental), essentially all cases of qīngchǔ in Chinese also.

Reply:
In Buck, there is a definition provided which shows that he means obskur and not undurchsichtig by obscure, and it's even in the chapter "mind, thought". But for the other ambiguous cases it makes a lot of sense to have those additional options.

Reply:
I'd propose the following definition for the concept CLEAR (VISUAL):

"If something is visible without obstacles (e.g., clouds, spots)."

Reply:
The general clear is, following [Collins](https://www.collinsdictionary.com/de/worterbuch/englisch/clear): Something that is clear is easy to understand, see, or hear. 

There is a second definition in Collins: Something that is clear is obvious and impossible to be mistaken about. 

But if we go along the line of mental states, we would have @AnnikaTjuka's definition for the visual part "CLEAR (TO SEE)" (my suggestion for the gloss), and now I have to go to the meeting... more later, or discuss and decide this withoutt me please.

Reply:
I've used the new concept "CLEAR (TO SEE)" for the list.